### ISSUE TITLE
App displaying old version info in Teams messages sent by Nagios

### ISSUE THREAD

**Issue by aucompbiker** [atc0005/send2teams#287]
Adam,

I recently updated the send2teams app hosted on every Military REACH server and our Nagios to the latest stable version, and the apps is working as intended except when it comes to displaying the app's version info displayed at the bottom of posts in Teams.

Here's an example Teams message Nagios posted this morning.
```
[6:31 AM] Nagios Notifications
      
PROBLEM Service Alert: "Disk check - space" for mr7.lib.auburn.edu is WARNING
Notification Type: PROBLEM
Service: Disk check - space
Host: Q/A Website instance
Address: 131.204.73.136
State: WARNING
Date/Time: Thu Mar 9 06:31:06 CST 2023
Additional Info:
`DISK WARNING - free space: /var/log 381 MB (20% inode=98%):`
Message delivered by send2teams (v0.9.2-0-gdfae0b9) at 2023-03-09T06:31:06-06:00 on behalf of Nagios
```
Is this a bug in the app or a conf setting I need to update in Nagios.

**Comment by atc0005** [comment]
@aucompbiker I'm going to guess a bug as I made a lot of refactoring changes recently with the build process. 

Can you confirm what version you deployed (based on what you downloaded) and whether it was a standalone binary or a package?

**Comment by atc0005** [comment]
After thinking about it, I guess you installed via package and left the old manually deployed version behind.

Run `which send2teams` and see what you get back. The packages install to the system paths.

**Comment by aucompbiker** [comment]
Actually, I'm still relying on a custom Bash script to install the app's standalone binary to the `/usr/local/sbin/` system path on each host.

Here's the output from running the `which` command on nagios.lib.
```
zrbdmas@nagios:~$ which send2teams
/usr/local/sbin/send2teams
``` 
And the binary's version info.
```
zrbdmas@nagios:~$ send2teams --version

send2teams v0.11.0
https://github.com/atc0005/send2teams
```

**Comment by aucompbiker** [comment]
**mr6.lib**
```zrbdmas@mr6:~$ which send2teams
/usr/local/sbin/send2teams
zrbdmas@mr6:~$ send2teams --version

send2teams v0.11.0
https://github.com/atc0005/send2teams
```

**mr7.lib**
```
zrbdmas@mr7:~$ which send2teams
/usr/local/sbin/send2teams
zrbdmas@mr7:~$ send2teams --version

send2teams v0.11.0
https://github.com/atc0005/send2teams
```

**mr8.lib**
```
zrbdmas@mr8:~$ which send2teams
/usr/local/sbin/send2teams
zrbdmas@mr8:~$ send2teams --version

send2teams v0.11.0
https://github.com/atc0005/send2teams
```

**Comment by atc0005** [comment]
> Actually, I'm still relying on a custom Bash script to install the app's standalone binary to the `/usr/local/sbin/` system path on each host.

@aucompbiker 

I didn't catch this when you posted earlier, but I see now that you mentioned the `/usr/local/sbin/` path.

Run this on each system where the tool is deployed:

`sudo find /usr -type f -name 'send2teams*'`

I'm thinking that you likely have multiple copies of the tool deployed.

Here is an example of how I've configured the command definitions in Nagios for that tool in the past:

https://github.com/atc0005/check-vmware/blob/master/contrib/nagios/etc/nagios-plugins/config/send2teams.cfg

It is likely that the older version is still present and called from `/usr/local/bin/send2teams`.

The same may apply to the other servers you noted also.

**Comment by aucompbiker** [comment]
I just checked each system the binary is installed on and found that each system hosts a copy of the binary in ```/usr/local/bin/``` and ```/usr/local/sbin/```.  

That's what I get for trying to blindly reuse my own code. 

I fixed the paths in custom deployment bash script for this binary, redeployed the binary, tested the binary on mr6.lib by running the _force_tomcat_redeployment_ script. 

Here's the output from the latest Teams message.

```
[12:11 PM] Web Dev (MR6) Notifications     
Tomcat restart on mr6 SUCCESSFUL
Tomcat is reporting success via curl connection test.
Message delivered by send2teams (v0.11.0) at 2023-03-10T12:11:09-06:00
```

Thank you for all the help.
